### ISSUE TITLE
Devcontainer Build on WSL (with code-insiders) - can't find Dockerfile - uses Windows directory format - Works in Release Code

### ISSUE THREAD

**Issue by lioncubs** [microsoft/vscode-remote-release#6681]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.67.0-insider - d31c2035e4444d7ecc6b04efc0759ad99fb9242d - x86
- OS Version: Windows 10 Enterprise 21H1 - On WSL 

Steps to Reproduce:

1. Created a NEW dev dir in WSL /home/ubuntu/git/testapp
2. Open Released version of code
```
ubuntu@mikeb-lt2:~/git/testapp$ code --version
1.66.2
dfd34e8260c270da74b5c2d86d61aee4b6d56977
x64
```

3. Created a .devcontainer (CPP + features docker-from-docker, kubectl-helm-minikube, git, python - BUT THIS COULD BE ANY simple devcontainer as well
4. rebuild/open in devcontainer in code - built OK.
5. Close devcontainer
6. Open Workspace and attempting to rebuild the devcontainer in code-insider fails with error:  "\home\ubuntu\git\testapp\.devontainer\Dockerfile" does not exist.
7. code-insiders version:

```
ubuntu@mikeb-lt2:~/git/testapp$ code-insiders --version
1.67.0-insider
d31c2035e4444d7ecc6b04efc0759ad99fb9242d
x64
```

8.  Was working fine as of last week Weds/Thursday.

devcontainer.json below:
```json
// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
// https://github.com/microsoft/vscode-dev-containers/tree/v0.231.6/containers/cpp
{
	"name": "C++",
	"build": {
		"dockerfile": "Dockerfile",
		// Update 'VARIANT' to pick an Debian / Ubuntu OS version: debian-11, debian-10, debian-9, ubuntu-21.04, ubuntu-20.04, ubuntu-18.04
		// Use Debian 11, Debian 9, Ubuntu 18.04 or Ubuntu 21.04 on local arm64/Apple Silicon
		"args": { 
			"VARIANT": "ubuntu-20.04"
		}
	},
	"runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],

	// Set *default* container specific settings.json values on container create.
	"settings": {},

	// Add the IDs of extensions you want installed when the container is created.
	"extensions": [
		"ms-vscode.cpptools",
		"ms-vscode.cmake-tools"
	],

	// Use 'forwardPorts' to make a list of ports inside the container available locally.
	// "forwardPorts": [],

	// Use 'postCreateCommand' to run commands after the container is created.
	// "postCreateCommand": "gcc -v",

	// Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
	"remoteUser": "vscode",
	"features": {
		"docker-from-docker": "latest",
		"kubectl-helm-minikube": "latest",
		"git": "latest",
		"python": "3.7"
	}
}

```

<img width="1037" alt="Code_-_Insiders_CowDLiH9Zd" src="https://user-images.githubusercontent.com/365203/166416275-d6b9dbbe-5ec2-43a2-aca4-33ca4c7a213c.png">

**Comment by chrmarti** [comment]
Fixed in Remote-Containers 0.233.1 pre-release. (https://github.com/microsoft/vscode-remote-release/issues/6679)

**Comment by lioncubs** [comment]
Verified Works for me in latest 👍
